frontend/catalyst/autograph.py

```diff
@@ -134,13 +122,96 @@ def autograph(fn):
     return new_fn
 
 
+def autograph_source(fn):
+    """Utility function to retrieve the source code of a function converted by AutoGraph.
```

Comment:
@dime10 the docs are failing to build:

```
AttributeError: module 'catalyst' has no attribute 'autograph_source'
```

Probably because of this line here,

https://github.com/PennyLaneAI/catalyst/blob/7781c683dc0a81ba95bb10d41816c7dceacde191/doc/dev/autograph.rst?plain=1#L751

where it is attempting to be imported by Sphinx for rending in the docs. Has the import location of this function changed?

Reply:
Replied in the other thread, but the module was removed and the function moved to a different module :)